benchmark/single-source/SortStrings.swift

```diff
@@ -1520,7 +1520,7 @@ let unicodeWords: [String] = [
   "❄️physicianless",
   "❄️bronchopathy",
   "❄️marabuto",
-  "❄️proprietory",
+  "❄proprietary",
```

Comment:
That doesn’t look right

Reply:
Feel free to make suggestions 😊

Reply:
It's an [alternative form](https://en.wiktionary.org/wiki/proprietory) of "proprietary", so you don't need to change it :).

Reply:
I meant it’s missing a character (it’s called a variation selector iirc) after the snowflake character.

Reply:
Please don't change anything in test strings in general unless it's clearly just logging. They might be the way they are for a reason.